modules/default/newsfeed/newsfeed.js

```diff
@@ -176,19 +180,23 @@ Module.register("newsfeed", {
 			});
 		}
 	},
-
+	getFeedProperty(feed, property) {
```

Comment:
I added this function so you can access values inside the objects in the feed array.